src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -148,7 +148,7 @@ class ClusterMonitorActor(val cluster: Cluster,
     } yield {
       // Finally pipe a shutdown message to this actor
       logger.info(s"Cluster ${cluster.googleProject}/${cluster.clusterName} is ready for use!")
-      ShutdownActor()
+      ShutdownActor(Some(RemoveFromList(cluster)))
```

Comment:
Is there ever a case now where we _don't_ pass a message to `ShutdownActor`? I wonder if we should make the parameter non-optional, and make `RemoveFromList(cluster)` the default.

Reply:
How would I make `RemoveFromList(cluster)` the default, since `RemoveFromList` needs an instance of `Cluster`?